multiple_generate.sh

```diff
@@ -4,17 +4,47 @@ function cloneSubProject() {
   git clone $1 $2 && cd $_
   rm -rf .git
   SUB_ROOT_DIR=$(pwd)
-  cd $3
-  if [ ! -f index.rst ] && [ ! -f index.md ]; then
-    echo -e "No index.rst found at $2/$3. Auto generating...\n"
-    python $BASE/modules/scripts/genindex.py $SUB_ROOT_DIR
-    if [ "${DEBUG:-false}" == "true" ]; then
-      echo -e "\n--------------------------------------\n"
-      cat index.rst
-      echo -e "\n--------------------------------------\n"
-    fi
-    echo -e "Auto generated index.rst\n"
+  # initializing the variables
+  export PROJECTNAME=""
+  export VERSION=""
+  export AUTHOR=""
+  export DEBUG=""
+  export MARKDOWN_FLAVOUR=""
+  export LOGO=""
+  export DOCTHEME=""
+  export DOCPATH=""
+  export AUTOAPI_PYTHON=""
+  export AUTOAPI_PYTHON_PATH=""
+  export AUTOAPI_JAVA=""
+  export AUTOAPI_JAVA_PATH=""
+  export MOCK_MODULES=""
+  export DOCURL=""
+  export HEROKU_APP_NAME=""
+  export SWAGGER_SPEC_URL=""
+  export SWAGGER_UI=""
+  export JAVADOC_PATH=""
+
+  ENVVARS="$(python ${BASE}/modules/scripts/config.py "${USERNAME}" "${REPONAME}")"
```

Comment:
Ok. Then I don't think this is a proper approach. This should be done from the webUI's end and not from the script's end. It would be preferable if from the webUI you just call the script in a loop and then call a seperate script that just aggregates the html from all the previews into a single folder.
@imujjwal96 Your views.